### PR TITLE
Created Templates for Feed Page

### PR DESCRIPTION
**Pull request by bryanansong**
## Description
I created the list item for my feed page. This shows relevant information about each template that a user can use to determine if they would like to copy that template. In future PRs I will make the list that is rendered come from the backend

<what this pull request is doing>
<what will be done in later PRs and not included here>

## Test Plan
Screenshot 2024-06-28 at 5 48 28 PM

<insert images or gifs of feature>
<otherwise, say how code was tested if not UI facing>
<any edge cases you might have specifically tested>

### CODE REVIEW

File: frontend/src/components/FeedListItem/FeedListItem.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import styles from "./FeedListItem.module.css";
+
+type FeedListItemProps = {
+  post: {
```
Comment (m5matthew) [inline frontend/src/components/FeedListItem/FeedListItem.tsx:4]: nit: In typescript, is it possible to create reusable types (eg. can we create a type called "FeedPost" and nest that type inside FeedListItemProps?) If possible, I would suggest doing that since we will probably have to reuse the FeedPost types in other components

**Review by m5matthew — COMMENTED** [review]
Styling looking really clean! Good stuff. For future UX polishes in later weeks, I think it would look really clean to add a copy icon next to "Copy template"

File: frontend/src/components/FeedListItem/FeedListItem.tsx (hunk 1 above)
Comment (bryanansong) [reply frontend/src/components/FeedListItem/FeedListItem.tsx:4]: I'm not aware if something like this exists. I will look into it
